webservice/src/services/ApiManager.js

```diff
@@ -31,6 +31,19 @@ export default class ApiManager {
     this.instanceNamespace = instanceNS;
   }
 
+  async retrieveImageList() {
+    return await this.apiCRD
+      .getClusterCustomObject(
+        'crownlabs.polito.it',
+        'v1alpha1',
+        'imagelists',
+        'crownlabs-virtual-machines'
```

Comment:
```suggestion
        'crownlabs-virtual-machine-images'
```
My fault, changed this name since the previous was less meaningful